src/modules/sign/icons/airbnblogo.js

```diff
@@ -1 +1,8 @@
-import React from 'react';
import airbnbLogo from "../../../assets/svg/airbnb.svg";

const AirbnbLogo = ({ className }) => {
  return <img src={airbnbLogo} alt="" className={className}/>
}

export default AirbnbLogo;
\ No newline at end of file
+import React from 'react';
+import airbnbLogo from "../../../assets/svg/airbnb.svg";
+
+const AirbnbLogo = ({ className }) => {
+  return <img src={airbnbLogo} alt="" className={className}/>
```

Comment:
Not a deal-breaker but you should try to provide meaningful values to image `alt` attributes. This helps with accessibility. You can read more about it [HERE](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML)